src/AggregateSpecsPlugin.ts

```diff
@@ -72,10 +80,9 @@ export const PgAggregatesSpecsPlugin: GraphileConfig.Plugin = {
             id: "sum",
             humanLabel: "sum",
             HumanLabel: "Sum",
-            isSuitableType: isNumberLike,
+            isSuitableType: isIntervalLike || isNumberLike,
```

Comment:
No idea how TypeScript allowed this... `function || function` is boolean, which shouldn't be allowed here.